INN/umbrella-sfpublicpress
# Link and button styles

## Changes

This pull request makes the following changes:

- Adds basic styling for inline links, basic links and default black and green buttons

Basic links:
![Screen Shot 2020-01-07 at 10 17 40 AM](https://user-images.githubusercontent.com/18353636/71905863-f6fd1e80-3136-11ea-8580-648d0a5cebee.png)

Inline links:
![Screen Shot 2020-01-07 at 10 10 26 AM](https://user-images.githubusercontent.com/18353636/71905328-f31ccc80-3135-11ea-8683-6e596d8c2ce1.png)

Buttons:
![Screen Shot 2020-01-07 at 10 10 06 AM](https://user-images.githubusercontent.com/18353636/71905329-f31ccc80-3135-11ea-8abd-93aa45dbd730.png)

## Why



For #9

## Testing/Questions

Features that this PR affects:

- Inline links
- All other `.entry-content a` links
- Buttons


Questions that need to be answered before merging:

- [ ] #9 was meant to style as many links and button types as possible, but I _think_ only inline links and the default black/green buttons (what's in this PR currently) are possible right now due to nothing else being built out. @benlk do you see anything else in the interactions design that could be styled right now?

Steps to test this PR:

1. View a post with an inline link and verify it looks ok
2. Add test buttons somewhere on the site with `.btn` and `.btn-green` classes and make sure they match the designs
